Dependencies/doorstop_config.ini

```diff
@@ -0,0 +1,3 @@
+[UnityDoorstop]
+enabled=true
+targetAssembly=QModManager/QModInstaller.dll
```

Comment:
Paths will have to be updated in both `QModManager` and `SMLHelper` to match the new folder locations